zip-draft-multisig-dev-fund.rst

```diff
@@ -0,0 +1,177 @@
+
+The implementation of this proposal would automatically direct 20% of newly issued ZEC in coinbase rewards between the first (year 2020, at block height 840,000) and second (year 2024, at block height 1,680,000) halving to the Zcash Development Fund. This allocation amounts to 1.05 million, or 5%, of fully diluted ZEC. Combined with the FR of 2.1 million, or 10%, of fully diluted ZEC, this amounts to 3.15 million, or 15%, of fully diluted ZEC.
+
+There are two crucial differences between the FR and the Zcash Development Fund. First, the Zcash Development Fund would be directed exclusively toward technical and non-technical initiatives that contribute to the success of Zcash. And second, to ensure proper accountability and transparency, the handling of the funds in the Zcash Development Fund would be subject to much more explicit and inclusive rules of governance, as laid out in the Specification section of this proposal.
+
+This proposal aspires to achieve the following:
+
+- To guarantee sufficient financial resources for research, development, maintenance, and other types of technical work directly connected to the various software implementations of the Zcash protocol by world-class cryptographers and engineers, as well as non-technical initiatives (including design, marketing, events, regulatory outreach, education, governance, and any other form of business or community development) that contribute to the long-term success of the Zcash network. Funding such activities constitutes the intended purpose of the Zcash Development Fund;
+
+- To ensure that funding for both technical and non-technical work on Zcash stays sufficiently independent from external entities (investors, donors, private companies, etc.) who could end up acquiring a disproportionately large influence over the network and its development, or jeopardize the sustainability of funding necessary for the success and stability of Zcash;
+
+- To establish the fundamental rules of governance and accountability regarding the use of funds in the Zcash Development Fund;
+
+- To increase the level of decentralization and community involvement in Zcash governance and resource allocation;
+
+- To encourage transparency and cooperation among different Zcash stakeholders and strengthen the community’s governance capabilities moving forward.
+
+Out of Scope
+============
+
+This proposal does not address the following closely related issues:
+
+- Details of the decision-making process for supporting or rejecting this or other relevant proposals by the ECC, the ZF, or other Zcash stakeholders. That said, the authors of this proposal maintain that any decision by the ECC and the ZF on the issue described in the Motivation section above MUST be preceded by at least the following procedures for measuring community sentiment, listed in the August 6, 2019 statement by the ZF [#ZFSTATEMENT]_:
```

Comment:
The second sentence and list cannot be a conformance requirement. In general, a ZIP can't impose conformance requirements on things it has described as out of scope (for reasons I hope are obvious, i.e. lack of clarity on whether it actually imposes the requirement or not) &mdash; even if there were not more fundamental issues of conflict with the current governance process in this particular case. If it is not removed, then wording like this would be acceptable:

> That said, the authors of this proposal believe it to be desirable for any decision by the ECC and the ZF on the issue described in the Motivation section above to be preceded by at least the following procedures for measuring community sentiment, listed in the August 6, 2019 statement by the ZF [#ZFSTATEMENT]_:

combined with removing RFC 2119 conformance language from the wording in the list (e.g. "Methods for measuring community sentiment could also ..."; and "The outcome ... should in our opinion play").